exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -159,7 +160,7 @@ def enable_ssh_access(container: Container, authorized_keys: str):
                 docker_db_image_info,
             )
 
-            ports = self._port_mappings()
+            self.port_forwards = self._port_mappings()
             volumes = {db_volume.name: {"bind": "/exa", "mode": "rw"}}
```

Comment:
this doesn't makes sense. you need the mapping only locally in this function. The attributes self.port_forwards was meant for the user supplied parameters and not for the mapping we generate from them.

Reply:
Will be removed in next push.